src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -387,7 +390,27 @@ private FcrepoResponse importContainer(final URI uri, final Model model) throws
             logger.debug("Using Bagit checksum ({}) for file ({})", checksum, containerFile.getPath());
             builder = builder.digest(checksum);
         }
-        return builder.preferLenient().perform();
+        final FcrepoResponse response = builder.preferLenient().perform();
+        if (response.getStatusCode() == 410 && config.overwriteTombstones()) {
+            deleteTombstone(response);
+            return builder.preferLenient().perform();
```

Comment:
I think this will work if the InputStream supports the `reset` method — which would be true for `ByteArrayInputStream` used and for `FileInputStream` (which is what the client would actually be using).  In theory, I could imagine future work that let you export from one Fedora and import into another in one step (without touching disk), and in that case, we'd need to support non-repeatable InputStreams like you get from HTTP responses.

I will add a test that uses a FileInputStream and has the binary be the resource with the tombstone, to make sure that the FileInputStream is actually repeatable.